### PR TITLE
Update instructions for adding a build

### PR DESCRIPTION
**Pull request by sravfeyn**
Updating docs

### CODE REVIEW

**Comment by sravfeyn** [comment]
@proteusvacuum

**Comment by sravfeyn** [comment]
Bumping for review. @proteusvacuum

File: corehq/apps/builds/README.rst [hunk 1]
```diff
@@ -103,53 +103,13 @@ You'll have to rebuild and redeploy your application to get it to sync.
...
-
-    ``python manage.py add_commcare_build $build_path $version $build_number``
-
-- Web UI
-
-  - Go to ``/builds/edit_menu/`` and follow the instructions at the bottom for adding your build.
-
-Now make sure the build is available in the app settings.  Go to ``/builds/edit_menu/``, then add the version and a label. You can also set the default here to be the version you've added.
+   #. In the Version field input the version in `x.y.z` format
+   #. In `Build Number` input any number (it doesn't matter what number)
```
Comment (proteusvacuum) [inline corehq/apps/builds/README.rst:110]: why does this field exist if it doesn't matter?

**Review by proteusvacuum — APPROVED** [review]

File: corehq/apps/builds/README.rst (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/builds/README.rst:110]: I am not really aware, but there might be some advanced use-case.

**Pull request merged**